packages/react-flight-dom-webpack/src/ReactFlightClientWebpackBundlerConfig.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+export type ModuleMetaData = {
+  id: string,
+  chunks: Array<string>,
+};
+
+type Thenable = {
+  then(resolve: () => mixed, reject: (mixed) => mixed): mixed,
+  ...
+};
+
+// The chunk cache contains all the chunks we've preloaded so far.
+// If they're still pending they're a thenable. This map also exists
+// in Webpack but unfortunately it's not exposed so we have to
+// replicate it in user space. null means that it has already loaded.
+const chunkCache: Map<string, null | Thenable> = new Map();
```

Comment:
@sokra In this approach, I'll always call `__webpack_chunk_load__` eagerly so there's always an opportunity for it to kick off async requests as a result. Always getting the same Promise back would be helpful since I wouldn't need to cache it. However, I also need to know if they've all been resolved so I know it's safe to call require.

This means I have to replicate this information in a user space map anyway.

It would be nice if there was a way to call `__webpack_chunk_load__`, so that there's something that spawns async work to kick off but synchronously know if it's already safe to call `__webpack_require__`. It's fine that we're forced to call `__webpack_chunk_load__` and even that it's always async, but if we have already done so in the past, it would be nice to query the webpack cache rather than our own.

Reply:
```suggestion
const chunkCache: Map<string | number, null | Thenable> = new Map();
```

I consider adding an extra argument to `__webpack_chunk_load__` which make it return `null` instead of a resolved Promise when the chunk is already loaded.

Reply:
That would be great! Thank you.